README.md

```diff
@@ -1,3 +1,4 @@
+#Objective: This lab will teach you how to implement nested forms. In this example, we will nest tags under posts, so a user can submit a post that has many tags.
```

Comment:
Can you make these Objectives in a bulleted fashion. Something [like this](https://github.com/learn-co-curriculum/iterators-and-abstraction-ruby#objectives)